x/vote/exported/types.go

```diff
@@ -43,6 +52,10 @@ func (m PollMetadata) Validate() error {
 		return fmt.Errorf("completed poll must have result set")
 	}
 
+	if m.Is(Completed) == (m.CompletedAt <= 0) {
+		return fmt.Errorf("completed poll must have completed at set")
```

Comment:
The error isn't appropriate if the poll is not completed but has a non-zero completed at value.

Reply:
"... and vice versa"?